cluster/src/test/java/com/hartwig/pipeline/storage/RuntimeBucketTest.java

```diff
@@ -133,6 +133,19 @@ public void namespacesListOperationPrefix() {
         assertThat(listOptionCaptor.getValue()).isEqualTo(Storage.BlobListOption.prefix("test/prefix"));
     }
 
+    @Test
+    public void usesCmekWhenProvided() {
+        RuntimeBucket.from(storage, NAMESPACE, referenceRunMetadata(), Arguments.testDefaultsBuilder().cmek("key").build());
+        assertThat(bucketInfo.getValue().getDefaultKmsKeyName()).isEqualTo(
+                "projects/hmf-pipeline-development/locations/europe-west4/keyRings/hmf-pipeline-development/cryptoKeys/key");
```

Comment:
Do we want to parameterise this? Makes the test somewhat brittle if we don't.

Reply:
Struggle a bit with this, as if I parameterize it, I basically end up rewriting the code I wrote in the RuntimeBucket itself. But maybe that's still better than fixing a million littles tests if the Arguments.testDefaults() change. I'll implement your suggestion